osquery/tables/system/linux/systemd_units.cpp

```diff
@@ -27,6 +27,7 @@ const std::vector<PropertyQueryDesc> kStringPropertyQueryList = {
     {"FragmentPath", "fragment_path", "org.freedesktop.systemd1.Unit"},
     {"SourcePath", "source_path", "org.freedesktop.systemd1.Unit"},
     {"User", "user", "org.freedesktop.systemd1.Service"},
+    {"UnitFileState", "enablement_state", "org.freedesktop.systemd1.Unit"},
```

Comment:
Maybe this should be called `unit_state`? Not sure whether that's specific enough.

cc @zhumo

Reply:
Technically this is about the unit file, not the unit, which are different objects in systemd (e.g. you'd normally list units with `ListUnits` but unit files with `ListUnitFiles`).
`unit_state` isn't very specific but I do think that `enablement_state` is a bit strange. I couldn't think of anything clearer.

Reply:
Why not just `unit_file_state`?

Reply:
I like Stefano's suggest `unit_file_state`. It's not succinct, but it matches what's there.

Reply:
```suggestion
    {"UnitFileState", "unit_file_state", "org.freedesktop.systemd1.Unit"},
```